exporter/otlpexporter/factory_test.go

```diff
@@ -178,13 +179,17 @@ func TestCreateTracesExporter(t *testing.T) {
 			factory := NewFactory()
 			creationParams := component.ExporterCreateParams{Logger: zap.NewNop()}
 			consumer, err := factory.CreateTracesExporter(context.Background(), creationParams, &tt.config)
-
-			if tt.mustFail {
+			if tt.mustFailOnCreate {
 				assert.NotNil(t, err)
 			} else {
```

Comment:
Not your fault, return on the positive branch so remove unnecessary indentation and else branch.

Reply:
done!